cmd/di.go

```diff
@@ -274,7 +275,7 @@ func (di *Dependencies) bootstrapNodeComponents(nodeOptions node.Options) {
 	di.ConnectionRegistry = connection.NewRegistry()
 	di.ConnectionManager = connection.NewManager(
 		dialogFactory,
-		payment_factory.PaymentIssuerFactoryFunc(nodeOptions, di.SignerFactory),
+		payment_factory.PaymentIssuerFactoryFunc(nodeOptions, di.SignerFactory, time.Second*20),
```

Comment:
I would do it as a constant in payments package, would also hint naming for some number.